doc/dev/custom_devices.rst

```diff
@@ -172,3 +177,117 @@ After doing so, Catalyst should be able to interface with your custom device.
     @qml.qnode(CustomDevice(wires=1))
     def f():
         return measure(0)
+
+
+Below is an example configuration file
+
+.. code-block:: toml
+
+        schema = 1
+
+        [device]
+        name = "dummy.device.qubit"
+        precision = ['float32', 'float64']
+
+        [operations]
+        # Observables supported by the device
+        observables = [
```

Comment:
Do you want to change the name of obs to their PL equivalents?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/369/commits/6b058e9bd6d1445ad354f988d72e391dab30273e